Agreements/External/AAlow_E.txt

```diff
@@ -1,2 +1,11 @@
+FIR_COPX:EHAM:*:ABNED:*:*:EHAA low:London AC Clacton:24000:*:ABNED
+FIR_COPX:EHAM:*:MIMVA:*:*:EHAA low:London AC Clacton:24000:*:MIMVA
+FIR_COPX:EHAM:*:KOLAG:*:*:EHAA low:London AC Clacton:24000:*:KOLAG
+
```

Comment:
Also EHAM-EGSS tfc and EHAM-EGKK tfc should be allowed via NOGRO and GALSO respectively.

Reply:
Will amend! And agreed levels diagrams too then?

Reply:
I guess, or put one arrow on the diagram implying all three fixes?

Reply:
I don't think EHAM departures inbound EGKK are permitted via GALSO? Level capping at FL240 and GALSO is for KK/KB/LC/MC only if RFL 260+?

EGKK high level should use GALSO and low level ABNED (low level is covered by first agreement in this file)

Added agreements for EHAM departures inbound SS/SC/GW/SH, routing via NOGRO.